app/managers.py

```diff
@@ -0,0 +1,44 @@
+import sqlite3
+
+from models import Actor
+
+
+class ActorManager:
```

Comment:
The class name 'ActorManager' is correctly in singular form, which aligns with the checklist requirement.